CODE_OF_CONDUCT.md

```diff
@@ -1,78 +1,85 @@
-* Public or private harassment
-* Publishing others' private information, such as a physical or electronic
- address, without explicit permission
+
+Examples of unacceptable behavior include but are not limited to:
+
+* The use of sexualized language or imagery
+* Trolling, insulting or derogatory comments, and personal or political attacks
+* Public or private harassment in any form
+* Publishing others' private information, such as a physical or email
+  address, without their explicit permission
+* Violence, threatening violence, or encouraging others to engage in violent behavior
+* Stalking or following someone without their consent
+* Unwelcome physical contact
+* Unwelcome sexual or romantic attention or advances
 * Other conduct which could reasonably be considered inappropriate in a
- professional setting
+  professional setting
 
-## Our Responsibilities
+The following behaviors are also prohibited:
 
-Project maintainers are responsible for clarifying the standards of acceptable
-behavior and are expected to take appropriate and fair corrective action in
-response to any instances of unacceptable behavior.
+* Providing knowingly false or misleading information in connection with a Code of Conduct investigation or otherwise intentionally tampering with an investigation.
+* Retaliating against a person because they reported an incident or provided information about an incident as a witness.
 
-Project maintainers have the right and responsibility to remove, edit, or
-reject comments, commits, code, wiki edits, issues, and other contributions
-that are not aligned to this Code of Conduct, or to ban temporarily or
-permanently any contributor for other behaviors that they deem inappropriate,
-threatening, offensive, or harmful.
+Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct.
+By adopting this Code of Conduct, project maintainers commit themselves to fairly and consistently applying these principles to every aspect
+of managing a CNCF project.
+Project maintainers who do not follow or enforce the Code of Conduct may be temporarily or permanently removed from the project team.
 
-## Scope
+## Reporting
 
-This Code of Conduct applies both within project spaces and in public spaces
-when an individual is representing the project or its community. Examples of
-representing a project or community include using an official project e-mail
-address, posting via an official social media account, or acting as an appointed
-representative at an online or offline event. Representation of a project may be
-further defined and clarified by project maintainers.
+For incidents occurring in the Metal3 community, contact the `Metal3 maintaner e-mail list` via <metal3-maintainers@googlegroups.com>. You can expect a response within three business days.
```

Comment:
As said, after re-reading the wording again, I don't think we're supposed to handle the CoC reports ourselves, but have CNCF handle them. This is clear distinction between the original CoC and CNCF CoC. Kubernetes here is just mentioned as an exception as it is such large project that they have separate CoC handling.

I also checked few other CNCF projects too. Not all of them implement CNCF CoC, but those who do did not change the CoC like this. This is of course up to discussion, but IMO we should keep the document as-is and not take the responsibility to ourselves to handle CoC, but leave it to CNCF.

Thoughts @kashifest ?

Reply:
I agree with @tuminoid's reasoning